zerver/openapi/zulip.yaml

```diff
@@ -9404,8 +9487,9 @@ paths:
                                 user_ids_string:
                                   type: string
                                   description: |
-                                    A string containing the ids of all users in the huddle(group PMs)
-                                    separated by commas(,). Example: "1,2,3".
+                                    A string containing the ids of all users in the group
```

Comment:
IDs should be capitalized that way, but this is preexisting with a few dozen violations we can sweep after merging this.
`$ git grep " ids " zerver/openapi/`

@laurynmm FYI, can you put such an `ids/IDs` sweep on your TODO list?  Not at all urgent and will merge conflict with this PR, so probably best to not do today :).

Reply:
Yes, :+1: on my list.